Makefile

```diff
@@ -71,4 +71,4 @@ test-race:
 	test/go-test-prettify -race ./...
 
 testpop:	clean
-	go build scripts/testpop/testpop.go
+	go build ./scripts/testpop/
```

Comment:
Since testpop now needs the other file in the testpop-directory (`statter.go`) as well, changed this to a relative path to the directory.